plugin/src/main/java/com/denizenscript/denizen/events/entity/EntityEntersVehicleScriptEvent.java

```diff
@@ -79,13 +79,11 @@ public ScriptEntryData getScriptEntryData() {
 
     @Override
     public ObjectTag getContext(String name) {
-        if (name.equals("vehicle")) {
-            return vehicle.getDenizenObject();
-        }
-        else if (name.equals("entity")) {
-            return entity.getDenizenObject();
-        }
-        return super.getContext(name);
+        return switch (name) {
+            case "vehicle" -> vehicle;

```

Comment:
Are you sure this doesn't need `getDenizenObject` it? What about stuff like rideable NPCs or a player mounting another player?